vendor/k8s.io/kubernetes/pkg/kubeapiserver/authenticator/patch_authenticator.go

```diff
@@ -0,0 +1,104 @@
+package authenticator
+
+import (
+	"fmt"
+	"time"
+
+	oauthclient "github.com/openshift/client-go/oauth/clientset/versioned"
+	oauthinformer "github.com/openshift/client-go/oauth/informers/externalversions"
+	userclient "github.com/openshift/client-go/user/clientset/versioned"
+	userinformer "github.com/openshift/client-go/user/informers/externalversions"
+	bootstrap "github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
+	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/authentication/group"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/keyutil"
+	"k8s.io/kubernetes/openshift-kube-apiserver/admission/authorization/restrictusers/usercache"
+	oauthvalidation "k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/oauth"
+	"k8s.io/kubernetes/openshift-kube-apiserver/authentication/oauth"
+	"k8s.io/kubernetes/openshift-kube-apiserver/enablement"
+	"k8s.io/kubernetes/pkg/serviceaccount"
```

Comment:
nit: split imports into openshift/kube groups

Reply:
> nit: split imports into openshift/kube groups

this is our file.  I really see no benefit to keeping them separate now that we have auto-importing capability (didn't exist 5 years ago)

Reply:
IMO it's a good habit to keep separate import contexts visibly separate, like syslibs/imports from elsewhere/imports from us